hosted_control_planes/hcp-deploy/hcp-deploy-bm.adoc

```diff
@@ -53,6 +53,22 @@ include::modules/hcp-bm-dns.adoc[leveloffset=+1]
 
 include::modules/hcp-custom-dns.adoc[leveloffset=+2]
 
+[id="hcp-bm-create-infra_{context}"]
+== Creating an infrastructure environment
+
+Before you can create a hosted cluster on bare metal, you need an infrastructure environment.
```

Comment:
s/infrastructure environment/infraenv/

InfraEnv it's an Assisted Installer object which is more specific, saying "infrastructure environment" sounds generic.